cmake/utils.cmake

```diff
@@ -121,3 +121,39 @@ function(get_target_property_list target outvar)
     endforeach()
     set(${outvar} ${${outvar}} PARENT_SCOPE)
 endfunction()
+
+# Some of our targets need to be linked for real. e.g. protoc, llvm-tlbgen and all its dependencies.
+macro(set_original_launchers_if_needed)
+    if(ENABLE_DUMMY_LAUNCHERS)
```

Comment:
If set_original_launchers_if_needed is called before set_dummy_launchers_if_needed we will overwrite CMAKE_C_COMPILER_LAUNCHER with zero string. Add a check that USING_DUMMY_LAUNCHERS is already set